pagopa/pn-frontend
# PN-13033 - Show notification cancelled document in timeline

## Short description
Show notification cancelled document in timeline

## List of changes proposed in this pull request
- Show document in timeline
- Enabled download for PF and PG

## How to test
- PA

1. Start the PA application and use "grossini / Comune di Palermo"
2. Open the notification with the IUN: `LPAK-MGXH-HRPJ-202411-M-1` and verify that the document titled `Dichiarazione annullamento notifica` is available for download.

- PF

1. Start PF and use "cristoforocolombo"
2. Open the notification with the IUN: `LPAK-MGXH-HRPJ-202411-M-1` and verify that the document titled `Dichiarazione annullamento notifica` is available for download.
